packages/smooth_app/lib/pages/product/product_image_gallery_view.dart

```diff
@@ -263,11 +266,16 @@ class _ProductImageGalleryViewState extends State<ProductImageGalleryView> {
           .whereNotNull();
 
   /// Created a [ProductImageData] from a [ProductImage]
-  ProductImageData _getProductImageData(ProductImage image) => ProductImageData(
-        imageField: image.field,
-        // TODO(VaiTon): i18n
-        title: image.imgid ?? '',
-        buttonText: image.imgid ?? '',
-        imageUrl: ImageHelper.buildUrl(widget.product.barcode, image),
-      );
+  ProductImageData _getProductImageData(ProductImage image) {
```

Comment:
You should put that as a factory/constructor in class `ProductImageData`, shouldn't you?